ScrumPoker.Business/GameRoomService.cs

```diff
@@ -45,4 +47,34 @@ public void DeleteById(int id)
     {
         _gameRoomRepository.DeleteById(id);
     }
+
+    public void AddPlayer(int gameRoomId, int playerId)
+    {
+        var gameRoomToAdd = _gameRoomRepository.GetById(gameRoomId);
+        var playerToAdd = _playerRepository.GetById(playerId);
+        gameRoomToAdd.Players.Add(playerToAdd);
+        
+        _gameRoomRepository.UpdatePlayerList(gameRoomToAdd);
+    }
+
+    public void RemovePlayer(int gameRoomId, int playerId)
+    {
+        var gameRoomToDeleteFrom = _gameRoomRepository.GetById(gameRoomId);
+        var playerToRemove = _playerRepository.GetById(playerId);
+
+        for (int i = 0; i < playerToRemove.GameRooms.Count; i++)
+        {
+            if(playerToRemove.GameRooms[i].Id.Equals(gameRoomToDeleteFrom.Id))
+                playerToRemove.GameRooms.RemoveAt(i);
+        }
+
+        for (int i = 0; i < gameRoomToDeleteFrom.Players.Count; i++)
+        {
+            if(gameRoomToDeleteFrom.Players[i].Id.Equals(playerToRemove.Id))
+                gameRoomToDeleteFrom.Players.RemoveAt(i);
+        }
```

Comment:
In general where possible we want to avoid index iteration if we can use linq.
When we have a complex data structure iterating and using indexes can be very hard to read.

From comment above this code should go away anyway, but here's an example using Linq
```
        var player = gameRoomToDeleteFrom.Players.Single(p => p.Id == playerId);
        gameRoomToDeleteFrom.Players.Remove(player);

        var gameRoom = player.GameRooms.Single(g => g.Id == gameRoomId);
        player.GameRooms.Remove(gameRoom);
```